scripts/inject-npm-version.js

```diff
@@ -25,13 +28,18 @@ const getBinaryVersion = async () => {
 }
 
 const parseSemanticReleaseOutput = (output) => {
+  debug(`Semantic Release Output ${output}`)
   const currentVersion = (output.match(/associated with version (\d+\.\d+\.\d+-?\S*)/) || [])[1]
   const nextVersion = (output.match(/The next release version is (\d+\.\d+\.\d+-?\S*)/) || [])[1]
 
+  debug(`Current Version`, currentVersion)
+  debug(`Next Version`, nextVersion)
```

Comment:
This could seem like an error when there is no next version to be released and it shows undefined (which is a majority of the time).